### ISSUE TITLE
Switch to vehicle map style when planning a driving route

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#6861]
And switch back to the previous style (outdoors or general) when changing the mode of transport in route planning.

**Comment by pastk** [comment]
Sounds logical first, but it'll make harder to find/set destination and intermediate points and explore what's there along the built route - because the navigation/vehicle style is much less detailed and doesn't display a lot of POIs.

**Comment by biodranik** [comment]
True. Looks like we need something intermediate for planning, with general style POIs, but with properly highlighted/colored car roads including their toll/surface/significance status and all necessary shields.
